e2e/run_e2e.sh

```diff
@@ -70,3 +70,22 @@ else
     echo "default-rb-cluster1 rolebinding not found"
     exit 1
 fi
+
+echo "TEST ClusterPermission with users and groups"
+kubectl config use-context kind-hub
+kubectl apply -f config/samples/clusterpermission_users_groups.yaml -n cluster1
+sleep 10
+work_kubectl_command=$(kubectl -n cluster1 get clusterpermission clusterpermission-users-groups -o yaml | grep kubectl | grep ManifestWork)
+if $work_kubectl_command; then
+    echo "ManifestWork found"
+else
+    echo "ManifestWork not found"
+    exit 1
+fi
+
+if kubectl -n kube-system get rolebinding kubevirt-rb-cluster1-users1; then
+    echo "kubevirt-rb-cluster1-users1 rolebinding found"
+else
+    echo "kubevirt-rb-cluster1-users1 rolebinding not found"
+    exit 1
+fi
```

Comment:
Also needs to check if `users1` and `users2` are the subjects right? Maybe:

```
if kubectl -n kube-system get rolebinding kubevirt-rb-cluster1-users1 -o yaml | grep users1; then
    echo found blah blah
else
...
if kubectl -n kube-system get rolebinding kubevirt-rb-cluster1-users1 -o yaml | grep users2; then
    echo found blah blah
else
....
```

if you don't to do any fancy kubectl commands.